timemachine/parallel/client.py

```diff
@@ -222,6 +226,10 @@ def verify(self):
         """
         gpus = get_gpu_count()
         assert self.max_workers <= gpus, f"More workers '{self.max_workers}' requested than GPUs '{gpus}'"
+        assert (
+            len(self._gpu_list) >= self.max_workers
+        ), "Fewer available GPUs than max workers expects, check CUDA_VISIBLE_DEVICES"
```

Comment:
I might be confused, but wouldn't this occur when `CUDA_VISIBLE_DEVICES` lists more GPUs than the `max_workers` given to the constructor? (Doesn't seem related to available GPUs?)